src/composables/queries/usePoolsGaugesQuery.ts

```diff
@@ -0,0 +1,100 @@
+import { useQuery, UseQueryOptions } from '@tanstack/vue-query';
+
+import QUERY_KEYS from '@/constants/queryKeys';
+import { subgraphRequest } from '@/lib/utils/subgraph';
+import { configService } from '@/services/config/config.service';
+
+/**
+ * TYPES
+ */
+
+type PoolsGaugesQueryResponse = Record<'pools', PoolsGauges[]>;
+export type PoolsGauges = {
+  preferentialGauge: {
+    id: string | null;
+  };
+  gauges: {
+    id: string;
+    relativeWeightCap: string;
+  }[];
+  address: string;
+  id: string;
+};
+
+type QueryOptions = UseQueryOptions<PoolsGaugesQueryResponse>;
+
+/**
+ * Fetches all gauges for a given pool and specifies which gauge is the
+ * preferential gauge.
+ */
+export default function usePoolsGaugesQuery(
```

Comment:
I think this might not be needed, see previous comment. If so, then we can delete.

Reply:
yes, removdd